client/state/checklist/actions.js

```diff
@@ -47,3 +49,49 @@ export const requestSiteChecklistTaskUpdate = ( siteId, taskId ) => ( {
 	siteId,
 	taskId,
 } );
+
+/**
+ * Returns true if site checklist notification is currently showing.
+ *
+ * @param {Object} state Global state tree
+ * @return {bool} True if currently showing checklist notification
+ */
+export function getChecklistStatus( state ) {
+	return state.checklist.showChecklistNotification;
+}
+
+/**
+ * Action creator function: SITE_CHECKLIST_NOTIFICATION
+ *
+ * @param {Boolean} bool Checklist notification
+ * @return {Object} action object
+ */
+export function setChecklistStatus( bool ) {
```

Comment:
The function name is too vague, isn't it? When I saw it at the very first time, I thought it would change whether a checklist item completes or not because we already use a variable named `taskStatuses` to indicate the status.